.npmrc

```diff
@@ -0,0 +1 @@
+node-linker=hoisted
```

Comment:
![image](https://user-images.githubusercontent.com/9800850/208896863-4f5fc40d-879d-4d38-aa08-1eea9ce46dfe.png)

🤣 

This kinda sums up most of my experience with non-classic `node_modules` 😅 

out of curiosity - what kind of an effect does it have on install times with pnpm?